integration-tests/tests/update_worker_set.rs

```diff
@@ -283,3 +289,140 @@ fn worker_set_update_can_be_resigned() {
         multisig_prover::msg::ProofStatus::Completed { .. }
     ));
 }
+
+#[test]
+fn register_active_workerset_with_monitoring_success() {
```

Comment:
why does this need it's own test case? Isn't this the exact same test as the first test in this file (worker_set_can_be_initialized_and_then_manually_updated)? The only difference seems to be that we check monitoring has the correct worker set. If so, just add that line into the first test